tests/models/whisper/test_modeling_tf_whisper.py

```diff
@@ -893,7 +891,7 @@ def test_large_batched_generation(self):
 
         # fmt: off
         EXPECTED_TRANSCRIPT = [
-            ' Mr. Quilter is the apostle of the middle classes, and we are glad to',
+            ' Mr. Quilter is the apostle of the middle classes and we are glad to',
```

Comment:
How come this has changed for the TF model, but not the PyTorch one?

Reply:
The pytorch already has that modification :) !